src/libmongoc/tests/unified/operation.c

```diff
@@ -2154,6 +2154,50 @@ operation_close (test_t *test,
    return ret;
 }
 
+static bool
+operation_drop_index (test_t *test,
+                      operation_t *op,
+                      result_t *result,
+                      bson_error_t *error)
+{
+   bool ret = false;
+   entity_t *entity;
+   bson_parser_t *parser = NULL;
+   char *index = NULL;
+   bson_t *opts = NULL;
+   mongoc_collection_t *coll = NULL;
+   bson_error_t op_error = {0};
+
+   parser = bson_parser_new ();
+   bson_parser_allow_extra (parser, true);
```

Comment:
```suggestion
```

IMO it is preferable not to allow extra fields in operations parsed from the specification test files. Unless those extra fields are used as an input for something else. Disallowing extra fields will immediately inform us if a spec test file adds an unexpected argument.